netci.groovy

```diff
@@ -71,7 +71,7 @@ static void addUnitPublisher(def myJob) {
       'xunit'('plugin': 'xunit@1.97') {
       'types' {
         'XUnitDotNetTestType' {
-          'pattern'('**/*TestResults.xml')
+          'pattern'('**/xUnitResults/*.xml')
```

Comment:
Won't this be different between our public and internal tests?  

Alos I'd separate this out. We already need to regenerate jobs today for other changes we're making.  Lets combine it all together and do it at once.